buildbot/dependency.conf

```diff
@@ -7,8 +7,8 @@ ocl_cpu_rt_ver_win=2020.11.8.0.27
 # https://github.com/intel/compute-runtime/releases/tag/20.35.17767
 ocl_gpu_rt_ver=20.35.17767
 # Same GPU driver supports Level Zero and OpenCL:
-# https://downloadmirror.intel.com/29817/a08/igfx_win10_100.8673.zip
-ocl_gpu_rt_ver_win=27.20.100.8673
+# https://downloadmirror.intel.com/29817/a08/igfx_win10_100.8778.zip
```

Comment:
I got https://www.intel.com/content/www/us/en/404.html when try to download RT
Please fix the link.

Reply:
Hi Vladimir, it is expected. The Windows GPU RT 8778 is not published yet. The ETA is WW40.2